modules/extension/graph/src/test/java/org/geotools/graph/util/DijkstraShortestPathFinderTest.java

```diff
@@ -126,6 +128,141 @@ public void test_2() {
       }  
     }
   }
+  
+  /**
+   * This tests the functionality of the NodeWeighter with equal node cost.
+   * 
+   * The test creates a graph with 4 nodes that consists of a single bifurcation.
+   * <BR><BR>
+   * 
+   * The shortest path is via the bifurcation (4 nodes), this is tested in the first stage. 
+   * In the second stage the bifurcation path is forbidden by increasing the node cost for the bifurcation node 
+   * if the outgoing edge is towards the bifurcation (new path consists of 99 nodes).
+   * 
+   * <BR><BR>
+   * Expected: <BR>
+   * 1. The node cost should be 0 for the source, 1 for the bifurcation and 3 for the two leafs.
+   */
+  public void test_3() {
+	    int nnodes = 4;
+	    
+	    // Creating a single bifurcation graph.
+	    Object[] singleBif = GraphTestUtil.buildSingleBifurcation(builder(), nnodes, 1);
+	    
+	    // Getting the source.
+	    Node source = (Node) singleBif[0];
```

Comment:
we use java code formatting with 100 cols width, this includes checking that spaces are used (rather than tabs).

For more information see our developers guide http://docs.geotools.org/latest/developer/conventions/code/code.html under coding style :)